viral/run_oasis.py

```
@@ -73,216 +75,93 @@ def modwt_denoise(
+        print("Using grosmark preprocessing")
+        spikes, denoised = grosmark_preprocess(s2p_path, plot)
 
-def _process_cell_no_plot_with_index(
-    args: tuple[int, np.ndarray, WheelFreeze],
-) -> tuple[int, np.ndarray, np.ndarray]:
-    """Driver for parallel processing, ensure cells are returned in the correct order"""
-    idx, cell, wheel_freeze = args
-    from pathlib import Path  # Needed for process_cell signature in subprocesses
-    import numpy as np
+    else:
+        dff = compute_dff(get_f(s2p_path))
```

Ruff (0.13.1)

Undefined name `compute_dff`

(F821)